tests/openvino/test_seq2seq.py

```diff
@@ -496,13 +496,15 @@ class OVModelForVisualCausalLMIntegrationTest(unittest.TestCase):
         SUPPORTED_ARCHITECTURES += ["gemma3", "smolvlm"]
     if is_transformers_version(">=", "4.51"):
         SUPPORTED_ARCHITECTURES += ["llama4"]
+    if is_transformers_version("<", "4.51"):
+        SUPPORTED_ARCHITECTURES += ["minicpmo"]
```

Comment:
Can you please also add a tests cases similar to these ones?

https://github.com/huggingface/optimum-intel/blob/main/tests/openvino/test_exporters_cli.py#L725-L734
https://github.com/huggingface/optimum-intel/blob/main/tests/openvino/test_quantization.py#L991-L1008